osu.Game/Overlays/Profile/Header/Components/OverlinedTotalPlayTime.cs

```diff
@@ -31,7 +32,7 @@ private void load(OverlayColourProvider colourProvider)
         {
             InternalChild = info = new OverlinedInfoContainer
             {
-                Title = "Total Play Time",
+                Title = UsersStrings.ShowStatsPlayTime,
```

Comment:
Tooltip for this component wasn't localized as it may require introducing a formattable localisable string type which supports plural variants (as per the web translation https://github.com/ppy/osu-resources/blob/5671764a75dd4ad6ef7275cdc649aa96a1ce5bb9/osu.Game.Resources/Localisation/Web/CommonStrings.cs#L257-L260)

Reply:
Note that different languages may have more than two forms (not only singular/plural), so this in general needs wider support and investigation how crowdin does this.

Reply:
Refer to the follow for how it's done:
- https://github.com/laravel/framework/blob/f60d3da8f9ad236d339562cd89eb5dfde4eb7dce/src/Illuminate/Translation/MessageSelector.php#L99
- https://discord.com/channels/188630481301012481/188630616286167041/853930675446218783